rskj-core/src/test/java/co/rsk/peg/BridgeSupportTest.java

```diff
@@ -5096,11 +5096,15 @@ public void getTransactionType_migration_tx() {
         }
         migrationTxInput.setScriptSig(inputScript);
         Assert.assertEquals(BridgeSupport.TxType.MIGRATION, bridgeSupport.getTransactionType(migrationTx));
+
+        when(mockFederationSupport.getRetiringFederation()).thenReturn(null);
+        when(provider.getLastRetiredFederationP2SHScript()).thenReturn(retiringFederation.getP2SHScript());
+        Assert.assertEquals(BridgeSupport.TxType.MIGRATION, bridgeSupport.getTransactionType(migrationTx));
     }
 
```

Comment:
Add tests for updateCollections now with `updateFederationCreationBlockHeights`. See `callProcessFundsMigration_is_migrating_after_rskip_146_activation` for example

Reply:
done